gtda/homology/simplicial.py

```diff
@@ -581,7 +582,7 @@ def plot(Xt, sample=0, homology_dimensions=None, plotly_params=None):
 @adapt_fit_transform_docs
 class WeakAlphaPersistence(BaseEstimator, TransformerMixin, PlotterMixin):
     """:ref:`Persistence diagrams <persistence_diagram>` resulting from
-    :ref:`weak alpha filtrations <TODO>`.
+    :ref:`weak alpha filtrations <weak_alpha_filtration>`.
```

Comment:
Am I right that this entry does not exist? This is for getting ready for its appearance, correct? I wonder if leaving TODO is better, because if the LaTeX entry ends up being called differently, we have to remember to change it. Leaving TODO means we can globally search for TODOs in our IDEs once in a while and not have false positives.

Reply:
I changed it for consistency. There are several entries (persistence_image, amplitude, finite_metric_spaces_and_point_clouds, weak_alpha_filtration) missing from the glossary and in the docstrings, they are all mentioned explicitly, instead of TODOs.
I am willing to change them all, if you think it would be better.

Reply:
Thanks for checking for consistency! Yes, for the reason I write above I do think that all inexistent links should be replaced by `TODO`. This way, whoever adds the glossary entry knows to replace any relevant `TODO` with the correct name.

Tagging @ammedmar as this might be of interest to him.